### PR TITLE
750/ Numpad

### PR DESCRIPTION
**Pull request by Mdemenko1**
## What (if any) features are you implementing?

Work for the issue: [750](https://github.com/Gary-Community-Ventures/benefits-calculator/issues/750)
Changing the keypad to numbers for the appropriate input fields. 
Inputs: Zip code, household size, your age, hours (if hourly frequency is selected), amount(all of them), additional person age, dollar amount

Ex:

image

### CODE REVIEW

File: src/Components/HouseholdDataBlock/HouseholdDataBlock.js [hunk 1]
```diff
@@ -108,6 +108,7 @@ const HouseholdDataBlock = ({ handleHouseholdDataSubmit }) => {
       inputName: 'age',
       inputValue: memberData.age,
       inputLabel: createFMInputLabel(personIndex),
+      numericField: true, // unsure where this one goes
```
Comment (CalebPena) [inline src/Components/HouseholdDataBlock/HouseholdDataBlock.js:111]: Did you mean to leave this comment?

File: src/Components/LoadingPage/LoadingPage.tsx [hunk 2]
```diff
@@ -16,7 +16,10 @@ const LoadingPage = () => {
 
   return (
     <main className="loading-wrapper">
-      <img src={loading} alt="loading-icon" className="spinner"></img>
+      <>
+        {/* <input type="text" pattern='\d*'/> */}
```
Comment (CalebPena) [inline src/Components/LoadingPage/LoadingPage.tsx:20]: Remove comment

File: src/Components/Textfield/Textfield.js [hunk 3]
```diff
@@ -34,6 +34,9 @@ const Textfield = ({ componentDetails, data, handleTextfieldChange, index, submi
       required
       error={errorController.showError}
       helperText={errorController.showError && errorController.message(data[inputName])}
+      // inputProps={numericField ? { pattern: "^\$\d{1,3}(,\d{3})*(\.\d+)?$",inputMode: 'decimal ' } : {}} //screenshot 
+      // inputProps={numericField ? { pattern: '\d*',inputMode: 'decimal ' } : {}}
```
Comment (CalebPena) [inline src/Components/Textfield/Textfield.js:38]: More comments to remove.

**Review by CalebPena — APPROVED** [review]
Looks good. There are a couple of comments that I think you did not mean to leave in. Also run `npm run format` to fix formatting.

File: src/Components/Textfield/Textfield.js [hunk 4]
```diff
@@ -34,6 +34,9 @@ const Textfield = ({ componentDetails, data, handleTextfieldChange, index, submi
...
+      inputProps={numericField ? { pattern:  '[0-9]* .' ,inputMode: 'decimal' } : {}}
```
[the remark this answers is not in the document]
Comment (Mdemenko1) [reply src/Components/Textfield/Textfield.js:39]: Thank you for pointing out this to me

**Pull request merged**